kedro/config/templated_config.py

```diff
@@ -113,24 +114,34 @@ def __init__(
                 dictionary. This dictionary will get merged with the globals dictionary
                 obtained from the globals_pattern. In case of duplicate keys, the
                 ``globals_dict`` keys take precedence.
+            anyconfig_kwargs: Keyword arguments to pass to the the ``anyconfig.load``
+                method. By default we enabled jinja2 via the ``ac_template`` argument.
         """
         super().__init__(
             conf_source=conf_source, env=env, runtime_params=runtime_params
         )
         self.base_env = base_env
         self.default_run_env = default_run_env
+        self._anyconfig_args = (
+            anyconfig_args if anyconfig_args else {"ac_template": True}
+        )
```

Comment:
Similarly you actually want something like `{"ac_template": True, **anyconfig_args}`.

Also I'm not sure about having `DEFAULT_ANYCONFIG_ARGS` in common.py while putting the defaults inside `__init__` here. I don't think which approach is best but I think we should probably do it the same way in both files.

Reply:
Since we flipped the booleaan on the other one they are different now...

Reply:
I also didn't know about this dict merging syntax, thought you had to go one level up